src/drc/api/scopes.py

```diff
@@ -5,10 +5,29 @@
 from vng_api_common.scopes import Scope
 
 SCOPE_DOCUMENTEN_ALLES_VERWIJDEREN = Scope(
-    'scopes.documenten.verwijderen',
+    'documenten.verwijderen',
     description="""
 **Laat toe om**:
 
 * documenten te verwijderen
 """
 )
+
+SCOPE_DOCUMENTEN_ALLES_LEZEN = Scope(
+    'documenten.lezen',
+    description="""
+**Laat toe om**:
+
+* documenten te lezen
+* documentdetails op te vragen
+"""
+)
+
+SCOPE_DOCUMENTEN_BIJWERKEN = Scope(
+    'documenten.bijwerken',
+    description="""
+**Laat toe om**:
+
+* attributen van een document te wijzingen
+"""
+)
```

Comment:
I think you forgot create

```suggestion
)

SCOPE_DOCUMENTEN_AANMAKEN = Scope(
    'documenten.aanmaken',
    description="""
**Laat toe om**:

* documenten aan te maken
```